### PR TITLE
Notebook

### LINKED ISSUE #430: Update Notebooks due to addition of predefined metrics
🐞 **Problem**
Now, there are predefined metrics available in symbolic module. Use them in example notebooks

🎯 **Goal**

  - [ ] [Notebook I](https://docs.einsteinpy.org/en/latest/examples/Playing%20with%20Contravariant%20and%20Covariant%20Indices%20in%20Tensors(Symbolic).html) - Replace the deprecated `SchwarzschildMetric`, with that from `predefined` module. 
  - [ ] [Notebook II](https://docs.einsteinpy.org/en/latest/examples/Ricci%20Tensor%20and%20Scalar%20Curvature%20symbolic%20calculation.html) - Use `AntiDeSitter` from `predefined` module, instead of explicitly defining it.

💡 **Possible solutions**

📋  **Steps to solve the problem**

 * Comment below about what you've started working on.
 * Add, commit, push your changes
 * Submit a pull request and add this in comments - `Addresses #<put issue number here>`
 * Ask for a review in comments section of pull request
 * Celebrate your contribution to this project 🎉

### PR DESCRIPTION
**Pull request by nimeshvashistha**
Fixes #430
apologies for closing the previous PR.
@ritzvik

### CODE REVIEW

File: docs/source/examples/Ricci Tensor and Scalar Curvature symbolic calculation.ipynb [hunk 1]
```diff
@@ -33,10 +34,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "syms = sympy.symbols(\"t chi theta phi\")\n",
-    "t, ch, th, ph = syms\n",
-    "m = sympy.diag(-1, cos(t) ** 2, cos(t) ** 2 * sinh(ch) ** 2, cos(t) ** 2 * sinh(ch) ** 2 * sin(th) ** 2).tolist()\n",
-    "metric = MetricTensor(m, syms)"
+    "AntiDeSitter().tensor()"
```
Comment (ritzvik) [inline docs/source/examples/Ricci Tensor and Scalar Curvature symbolic calculation.ipynb]:
```suggestion
    "metric = AntiDeSitter()"
```

Comment (ritzvik) [reply docs/source/examples/Ricci Tensor and Scalar Curvature symbolic calculation.ipynb]: Also in the last few lines of notebook add a line `R.simplify()` before `R.expr`

Comment (nimeshvashistha) [reply docs/source/examples/Ricci Tensor and Scalar Curvature symbolic calculation.ipynb]: yes alright

**Comment by ritzvik** [comment]
Thanks for this Nimesh, merging 🥂

File: src/einsteinpy/tests/test_geodesics.py [hunk 2]
```diff
@@ -44,7 +44,7 @@ def test_Geodesics_str_returns_members(dummy_data):
     geo = Geodesic(body, time=t, end_lambda=end_lambda, step_size=stepsize)
     assert (
         geo.__str__()
-        == "body name= (obj) , metric=(Schwarzschild) , parent name=(attractor) , parent mass=(6e+24 kg)"
+        == "Geodesic object body name= (obj) , metric=(Schwarzschild) , parent name=(attractor) , parent mass=(6e+24 kg)"
```
Comment (nimeshvashistha) [inline src/einsteinpy/tests/test_geodesics.py]: looks like this change got merged too. It is part of isssue #410.

File: src/einsteinpy/tests/test_geodesics.py [hunk 3]
```diff
@@ -53,5 +53,5 @@ def test_Geodesics_repr_returns_members(dummy_data):
     geo = Geodesic(body, time=t, end_lambda=end_lambda, step_size=stepsize)
     assert (
         geo.__repr__()
-        == "body name= (obj) , metric=(Schwarzschild) , parent name=(attractor) , parent mass=(6e+24 kg)"
+        == "Geodesic object body name= (obj) , metric=(Schwarzschild) , parent name=(attractor) , parent mass=(6e+24 kg)"
```
Comment (nimeshvashistha) [inline src/einsteinpy/tests/test_geodesics.py]: @ritzvik  This change as well.

File: src/einsteinpy/tests/test_geodesics.py (hunk 2 above)
Comment (shreyasbapat) [reply src/einsteinpy/tests/test_geodesics.py]: Alright, it seems fine to me. Will see tomorrow what issue needs to be closed.

File: src/einsteinpy/tests/test_geodesics.py (hunk 3 above)
Comment (ritzvik) [reply src/einsteinpy/tests/test_geodesics.py]: okay, no harm done :)

**Pull request merged**